src/main/webapp/app/lecture/lecture-unit/lecture-unit-management/attachment-units/attachment-units.component.html

```diff
@@ -99,7 +99,7 @@ <h2 id="page-heading">
             />
         </div>
     </div>
-    @if (units && units.length < 1) {
+    @if (!units || units!.length < 1) {
```

Comment:
```suggestion
    @if (units === null || units.length === 0) {
```

`x === null` equals to `x === null || x === undefined`.